runtime/compiler/optimizer/J9RecognizedCallTransformer.hpp

```diff
@@ -77,6 +77,22 @@ class RecognizedCallTransformer : public OMR::RecognizedCallTransformer
     *     \endcode
     */
    void process_java_lang_StringUTF16_toBytes(TR::TreeTop* treetop, TR::Node* node);
+   /** \brief
+    *     Transforms java/lang/StrictMath.sqrt(D)D into a CodeGen inlined function with equivalent semantics.
```

Comment:
Is there a reason we cannot handle `java/lang/[Strict]Math.sqrt(F)F` in a similar way?

Reply:
Is there such an overload? I don't see one in the java docs. The only one I see is for `double` type.

Reply:
You're right! TIL.